renderdoc/driver/vulkan/vk_acceleration_structure.cpp

```diff
@@ -68,6 +118,109 @@ void VkAccelerationStructureInfo::Release()
     delete this;
 }
 
+uint64_t VkAccelerationStructureInfo::GetSerialisedSize() const
+{
+  uint64_t geomDataSize = 0;
+  for(const GeometryData &geoData : geometryData)
+    geomDataSize += geoData.GetSerialisedSize();
```

Comment:
I think this would be better simplified as `geoData.byteSize()` which multiplies `sizeof` by count automatically.